src/components/scrollable-list/scrollable-list-item.component.js

```diff
@@ -0,0 +1,20 @@
+import styled, { css } from 'styled-components';
+import propTypes from 'prop-types';
+
+const StyledListItem = styled.li`
```

Comment:
As there is no distinction in this case between the component and the style, do you think the name of this should reflect the component name so it makes sense in dev tools etc? eg. `ScrollableListItem`.

You could achieve this be either renaming this `const` or adding `StyledListItem.displayName = 'ScrollableListItem';`